flatpak-tweaks.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/cmake/FilesystemResolver.cmake b/cmake/FilesystemResolver.cmake
+index acef652..1bd3a15 100644
+--- a/cmake/FilesystemResolver.cmake
++++ b/cmake/FilesystemResolver.cmake
+@@ -12,7 +12,7 @@ else()
+     set(USING_BOOST_FILESYSTEM FALSE PARENT_SCOPE)
+     set(USING_BOOST_FILESYSTEM FALSE)
+     message(STATUS "[FilesystemResolver]: Using standard C++ filesystem API")
+-    if(APPLE OR "${CMAKE_SYSTEM}" MATCHES "Linux")
++    if(APPLE)
+         set(FILESYSTEM_LIBS "stdc++fs")
```

Comment:
BTW this is a bug upstream whereas it assumes a specific version of ubuntu with an old version of gcc.